### PR TITLE
Add proteus abundance matrix to maxquant testdata

### PR DESCRIPTION
**Pull request by WackerO**
This PR adds the output from proteus::readproteingroups of this dataset

### CODE REVIEW

File: data/proteomics/README.md [hunk 1]
```diff
@@ -18,6 +18,22 @@ For MaxQuant, the files `proteinGroups.txt` and `experimentalDesignTemplate.txt`
...
 Finally, `MaxQuant_contrasts.csv` was created to be used as a contrast file for the [nf-core/differentialabundance pipeline](https://github.com/nf-core/differentialabundance). Contrasts were decided (for no particular reason) to be set up as comparisons between T1 cells and the other three cell types in the dataset; in the last comparison, the batch effect is also included. Additionally, in the last row, a contrast was created out of the fakeBatch column; this was done simply to allow for module/pipeline tests with multiple conditions.
 
+To test some downstream modules, the proteus R package was used to read the proteinGroups and create an abundance matrix. For this, on a MacBook Pro with a 2,4 GHz Quad-Core Intel Core i5 processor, in RStudio version 2023.06.2+561 and R version 4.2.3 (2023-03-15), the following script was run using Proteus version 0.2.16 in order to produce `proteus.raw_MaxQuant_proteingroups_tab.tsv`:
+
+'''
+library(proteus)
+sample.sheet <- read.table("MaxQuant_samplesheet.tsv", header=T, sep="\t")
```
Comment (mirpedrol) [inline data/proteomics/README.md:25]: Could we add a link to the original `MaxQuant_samplesheet.tsv` file?

File: data/proteomics/README.md [hunk 2]
```diff
@@ -18,6 +18,22 @@ For MaxQuant, the files `proteinGroups.txt` and `experimentalDesignTemplate.txt`
...
+sample.sheet$sample <- sample.sheet[["Experiment"]]
+sample.sheet$condition <- sample.sheet[["Celltype"]]
+measure.cols <- setNames(paste0("LFQ intensity ", sample.sheet[["Experiment"]]), sample.sheet[["Experiment"]])
+proteinGroups <- readProteinGroups(file="MaxQuant_proteinGroups.txt", meta=sample.sheet, measure.cols=measure.cols, data.cols=proteus::proteinColumns)
```
Comment (mirpedrol) [inline data/proteomics/README.md:29]: Add the reference to `MaxQuant_proteinGroups.txt` too

**Review by mirpedrol — APPROVED** [review]
I would add the reference to the files used to generate the new one. After that looks good to me :)

**Comment by WackerO** [comment]
Thanks Júlia, links are added!

**Pull request merged**